routing_model/index.html

```diff
@@ -197,14 +199,33 @@ <h2>Node as a member of a section</h2>
     InitializeNodeInternalState --> ConcurentStartElder
 
     ConcurentStartElder{"Concurrent<br/>start elder"}
-    ConcurentStartElder --> StartTopLevelDst
-    style StartTopLevelDst fill:#f9f,stroke:#333,stroke-width:4px
 
-    ConcurentStartElder --> StartTopLevelSrc
+    ConcurentStartElder --> ConcurentStartSrc
+    ConcurentStartSrc{"Concurrent<br/>start src"}
+
+    ConcurentStartSrc --> StartTopLevelSrc
     style StartTopLevelSrc fill:#f9f,stroke:#333,stroke-width:4px
 
-    ConcurentStartElder --> StartCheckAndProcessElderChange
-    style StartCheckAndProcessElderChange fill:#f9f,stroke:#333,stroke-width:4px
+    ConcurentStartSrc --> StartRelocateSrc
+    style StartRelocateSrc fill:#f9f,stroke:#333,stroke-width:4px
+
+
+    ConcurentStartElder --> ConcurentStartDst
+    ConcurentStartDst{"Concurrent<br/>start dst"}
+
+    ConcurentStartDst --> StartTopLevelDst
+    style StartTopLevelDst fill:#f9f,stroke:#333,stroke-width:4px
+
+    ConcurentStartDst --> StartRelocatedNodeConnection
+    StartRelocatedNodeConnection[Start RelocatedNodeConnection]
```

Comment:
`Start RelocatedNodeConnection` and `Start ResourceProof`:
These spaces are inconsistent with all other subroutines

Reply:
Yes, I'm not super happy with these names either. Fixed for now.